custom_components/opnsense/services.py

```
@@ -30,277 +31,576 @@
+    service_identifier = call.data.get("service_id", call.data.get("service_name"))
+    if not service_identifier:
+        raise _service_validation_error(_TRANSLATION_KEY_SERVICE_IDENTIFIER_REQUIRED)
+    return service_identifier
+
+
+async def _run_boolean_client_action(
+    clients: list[OPNsenseServiceClient],
+    log_prefix: str,
+    action_name: str | None,
+    target_name: str,
+    target_value: Any,
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `target_value`

(ANN401)